frontend/src/scenes/cohorts/Cohort.tsx

```diff
@@ -210,86 +202,261 @@ export function Cohort({ id }: { id?: CohortType['id'] } = {}): JSX.Element {
+                                                                    onCriteriaChange(newGroup, group.id)
+                                                                }
+                                                                onRemove={() => {
+                                                                    setCohort({
+                                                                        ...cohort,
+                                                                        groups: cohort.groups.filter(
+                                                                            (_, i) => i !== index
+                                                                        ),
+                                                                    })
                                                                 }}
-                                                            >
-                                                                {kids}
-                                                                <Row>
-                                                                    {error && (
-                                                                        <div
-                                                                            style={{
-                                                                                color: 'var(--danger)',
-                                                                                marginTop: 16,
-                                                                            }}
-                                                                        >
-                                                                            {error}
-                                                                        </div>
-                                                                    )}
-                                                                </Row>
-                                                            </div>
-                                                        )
+                                                                group={group}
+                                                                hideRemove={cohort.groups.length === 1}
+                                                            />
+                                                        </KeaField>
+                                                        {index < cohort.groups.length - 1 && (
+                                                            <div className="stateful-badge or mt mb">OR</div>
+                                                        )}
+                                                    </div>
+                                                </Group>
+                                            ))}
+                                            <span id="add" />
+                                            <div style={{ marginTop: 8, marginBottom: 8 }}>
```

Comment:
There's no real standard since use both liberally everywhere. Here I just opted for inline since it wasn't worth sticking another class into the scss for just some margins.